src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/PerformanceController.php

```diff
@@ -120,8 +133,18 @@ public function processFormAction(Request $request)
      */
     public function clearCacheAction()
     {
-        $this->get('prestashop.adapter.cache_clearer')->clearAllCaches();
-        $this->addFlash('success', $this->trans('All caches cleared successfully', 'Admin.Advparameters.Notification'));
+        if (!in_array(
+            $this->authorizationLevel($this::CONTROLLER_NAME),
+            array(
+                PageVoter::LEVEL_DELETE,
+            )
+        )) {
+            $this->addFlash('error', $this->trans('You do not have permission to update this.', 'Admin.Notifications.Error'));
```

Comment:
shouldn't be: "You do not have permission to delete this."

Reply:
If yes, use the same domains: Admin.Notifications.Error